pom.xml

```diff
@@ -99,17 +102,17 @@
         <jboss-logging.version>3.2.1.Final</jboss-logging.version>
         <seata.version>2.2.0</seata.version>
         
-        <netty.version>4.1.112.Final</netty.version>
-        <bouncycastle.version>1.78.1</bouncycastle.version>
+        <netty.version>4.1.117.Final</netty.version>
+        <bouncycastle.version>1.80</bouncycastle.version>
         
-        <curator.version>5.7.0</curator.version>
+        <curator.version>5.7.1</curator.version>
         <zookeeper.version>3.9.2</zookeeper.version>
-        <audience-annotations.version>0.12.0</audience-annotations.version>
+        <audience-annotations.version>0.15.0</audience-annotations.version>
         <jetcd.version>0.7.7</jetcd.version>
-        <vertx.version>4.5.1</vertx.version>
+        <vertx.version>4.5.11</vertx.version>
```

Comment:
- Shardingsphere doesn't actually use vertx. `io.vertx:vertx-grpc:4.5.1` is a transitive dependency of `io.etcd:jetcd-grpc:0.7.7`, why do we need to change the dependency tree of jetcd?